app/views/queries/_filters.html.erb

```diff
@@ -169,15 +168,15 @@ when  :list, :list_optional, :list_status, :list_subprojects %>
       <%= l(:label_filter_add) %>
     </label>
     <label for="add_filter_select" class="hidden-for-sighted">
-       {{ I18n.t('js.filter.description.text_open_filter') }}
-       {{ I18n.t('js.filter.description.text_close_filter') }}
+      {{ I18n.t('js.filter.description.text_open_filter') }}
+      {{ I18n.t('js.filter.description.text_close_filter') }}
     </label>
     <div class="advanced-filters--add-filter-value">
-       <%= select_tag 'add_filter_select',
-                      options_for_select(entries_for_filter_select_sorted(query)),
-                      onchange: "add_filter();",
-                      class: "advanced-filters--select",
-                      name: nil %>
+      <%= select_tag 'add_filter_select',
+                     options_for_select(entries_for_filter_select_sorted(query)),
+                     onchange: "add_filter();",
+                     class:    "advanced-filters--select",
+                     name:     nil %>
     </div>
   </li>
```

Comment:
Filters that are added are no longer (I just assume they were) active by default.